cmd/install/assets/recordingrules/hypershift.yaml

```diff
@@ -2,6 +2,15 @@ groups:
 - name: hypershift.rules
   interval: "30s"
   rules:
+  - record: hypershift:apiserver_request_total
+    expr: apiserver_request_total{}
```

Comment:
I think this is an extreme high carnality metric and we get aggregated versions of the data in the next two rules.  Why get the raw metric?